webui/apps/docs/src/content/docs/docs/howto/use_k8s.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Deploy Eidolon AI with Kubernetes
+description: Learn how to deploy and manage Eidolon AI in a Kubernetes environment
+---
+
+Running Eidolon AI on Kubernetes unlocks powerful deployment and scaling capabilities. This guide walks you through setting up Eidolon AI in your Kubernetes cluster.
+
+## Prerequisites
+
+- A running Kubernetes cluster and `kubectl` CLI ([Get started with Minikube][minikube])
+- Basic familiarity with [Eidolon AI](https://www.eidolonai.com/docs/)
+
+## Quick Setup
+
+1. Clone the quickstart repo:
+   ```bash
+   git clone git@github.com:eidolon-ai/eidolon-quickstart.git
+   ```
+
+2. Create a namespace (optional for Make users):
+   ```bash
+   kubectl create namespace eidolon-quickstart
+   ```
+
+3. Set up API access tokens using Kubernetes secrets:
+   ```bash
+   # Option 1: From .env file
+   kubectl create secret generic eidolon --from-env-file=.env
+
+   # Option 2: Direct configuration
+   kubectl create secret generic eidolon \
+     --from-literal=OLLAMA_URL=https://ollama.example.com \
+     --from-literal=OPENAI_API_KEY=your-openai-api-key \
+     --from-literal=MISTRAL_API_KEY=your-mistral-api-key \
+     --from-literal=ANTHROPIC_API_KEY=your-anthropic-api-key
```

Comment:
since the quickstart only uses openai, we can probably trim this to just OPENAI_API_KEY